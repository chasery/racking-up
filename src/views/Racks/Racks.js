import React from "react";
import RacksList from "../../components/RacksList/RacksList";
import "./Racks.css";

function Racks(props) {
  return (
    <main role="main">
      <h2>Racks</h2>
      <RacksList />
    </main>
  );
}

export default Racks;
