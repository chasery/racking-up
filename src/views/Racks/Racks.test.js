import React from "react";
import { shallow } from "enzyme";
import toJson from "enzyme-to-json";
import Racks from "./Racks";

describe("Racks component", () => {
  it("renders without error", () => {
    const wrapper = shallow(<Racks />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
