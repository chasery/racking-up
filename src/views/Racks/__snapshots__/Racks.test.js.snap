// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Racks component renders without error 1`] = `
<main
  role="main"
>
  <section
    className="Racks"
  >
    <div
      className="Racks__wrapper"
    >
      <h3>
        My Racks
      </h3>
      <RacksList />
    </div>
  </section>
</main>
`;
